kafka-companion-frontend/src/app/navbar/navbar.component.html

```diff
@@ -7,6 +7,7 @@
     <input class="kafka-search-input" type="text" placeholder="" (input)="onPhraseChange($event.target.value)" #searchInput>
   </div>
   <div class="kafka-spring"></div>
-  <a class="kafka-logo" [routerLink]="['/']"><span>Kafka Companion</span> <img class="kafka-logo-image" src="assets/apache-kafka.png"/></a>
+  <a class="kafka-logo" href="https://www.consdata.com/"> <img class="consdata-logo-image" src="assets/consdata-logo.png"/></a>
+  <a class="kafka-logo" href="https://github.com/Consdata/kafka-companion"><span>Kafka Companion v1.2</span> <img class="kafka-logo-image" src="assets/apache-kafka.png"/></a>
```

Comment:
bez tej wersji na razie. na to jest osobne zadanie

Reply:
fakt, poprawione :)